packages/kobotoolbox/src/Adaptor.js

```diff
@@ -51,53 +43,69 @@ export function execute(...operations) {
  */
 export function getForms() {
   return async state => {
-    const url = `/assets/?asset_type=survey`;
+    const response = await util.request(state, 'GET', 'assets', {
+      query: { asset_type: 'survey' },
+    });
 
-    const response = await util.request(state, 'GET', url, {});
+    console.log('✓', response.body.results?.length, 'forms fetched.');
 
-    console.log('✓', response.body.results.length, 'forms fetched.');
     return util.prepareNextState(state, response);
   };
 }
 
 /**
  * Get submissions for a specific form. Calls `/api/v2/assets/<formId>/data/`.
  * @example <caption>Get all submissions for a specific form</caption>
- * getSubmissions('aXecHjmbATuF6iGFmvBLBX');
+ * getSubmissions('aXecHjmbATuF6iGFmvBLBX', { limit: Infinity });
  * @example <caption>Get form submissions with a query</caption>
- * getSubmissions('aXecHjmbATuF6iGFmvBLBX', { query: { _submission_time:{ $gte: "2022-06-12T21:54:20" } } });
+ * getSubmissions('aXecHjmbATuF6iGFmvBLBX', { query: { _submission_time:{ $gte: "2025-03-12T21:54:20" } } });
  * @function
  * @public
  * @param {string} formId - Form Id to get the specific submissions
  * @param {object} [options={}] - Optional query params for the request
+ * @param {object} [options.query] - (Openfn only) Query options to filter the submissions. See query operators {@link http://docs.mongodb.org/manual/reference/operator/query/.}
+ * @param {number} [options.limit=10000] - (Openfn only) Maximum number of submissions to fetch.
```

Comment:
Here would be a good place to say `Pass Infinity to disable the limit and download all submissions"